frappe/desk/form/load.py

```diff
@@ -101,7 +102,8 @@ def get_docinfo(doc=None, doctype=None, name=None):
 		"energy_point_logs": get_point_logs(doc.doctype, doc.name),
 		"milestones": get_milestones(doc.doctype, doc.name),
 		"is_document_followed": is_document_followed(doc.doctype, doc.name, frappe.session.user),
-		"tags": get_tags(doc.doctype, doc.name)
+		"tags": get_tags(doc.doctype, doc.name),
+		"email_link": get_email_link(doc.doctype, doc.name)
```

Comment:
```suggestion
		"document_email": get_docuemnt_email(doc.doctype, doc.name)
```
? 
because it is not link yet.

Reply:
@surajshetty3416 The feature is called `automatic email link`, hence named the key as `email_link`